docs/.vuepress/components/Footer.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <div>
+        <div class='footer'>
+            <img alt='RLMM logo' src='icons/logo_rlmm_round_144.png'></img>
+            <a href='https://www.youtube.com/channel/UCp6gkqadtgsaZmPnex9MNiA/featured'>YouTube</a>
+            <a href='https://discord.gg/PWu3ZWa'>Discord</a>
+            <a href='https://github.com/RocketLeagueMapmaking/RL-docs'>GitHub</a>
+            <a href='./more/contact'>Contact</a>
+            <a href='./more/about'>About</a>
+        </div>
+
+        <div class='under-footer'>
+            <p>Powered by Vuepress   |   Written by Mr. Swaggles   |   Designed by <a href="#">ghostrider-05</a></p>
```

Comment:
Are you missing the link to your name? if not, remove it since it might be confusing